scripts/index.js

```diff
@@ -0,0 +1,42 @@
+function getRandom(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+let dieOne = {
+	value: 1,
+	roll: function() {
+		this.value = getRandom(1, 6);
+		return this.value;
```

Comment:
Well done, it is considered good practice to return a value even from object methods which mutate the object.